CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Some integration tests requires Docker to be installed.
 ## How to contribute
 
 Fork the repository, make changes, and send us a
-[pull request](https://www.yegor256.com/2014/04/15/github-guidelines.html). We will review
+[pull request](#pull-request). We will review
```

Comment:
@olenagerasimova it's a link to this document in Markdown format: `[text](ref)`, where `#sec` is a reference to the section in document

Reply:
@g4s8 yes, I see, that it is a link. I'm suggesting to introduce shortening for the "pull request" here instead of line 72, because now shortening "PR" is used before we actually introduce it (on line 68 for example).

Reply:
@olenagerasimova I see, fixed in a new commit